pop/service/api.go

```diff
@@ -144,6 +146,64 @@ func (c *Client) Merge(dst network.Address, p *PopDesc, priv kyber.Scalar) (
 	return res.Final, nil
 }
 
+// GetLink returns the link of the organizer, if available.
+func (c *Client) GetLink(dst network.Address) (kyber.Point, error) {
+	si := &network.ServerIdentity{Address: dst}
+	res := &GetLinkReply{}
+
+	e := c.SendProtobuf(si, &GetLink{}, res)
+	if e != nil {
+		return nil, e
+	}
+	return res.Public, nil
+}
+
+// GetFinalStatements returns a map of all final statements.
+func (c *Client) GetFinalStatements(dst network.Address) (map[string]*FinalStatement, error) {
+	si := &network.ServerIdentity{Address: dst}
+	res := &GetFinalStatementsReply{}
+
+	e := c.SendProtobuf(si, &GetFinalStatements{}, res)
+	if e != nil {
+		return nil, e
+	}
+	return res.FinalStatements, nil
+}
+
+// StoreInstanceID asks the service to store an instanceID for a given party.
+func (c *Client) StoreInstanceID(dst network.Address, partyID []byte, instanceID ol.InstanceID) error {
+	si := &network.ServerIdentity{Address: dst}
+	ret := &StoreInstanceIDReply{}
+
+	return c.SendProtobuf(si, &StoreInstanceID{partyID, instanceID}, ret)
+}
+
+// GetInstanceID asks the service for an instanceID for a given party.
+func (c *Client) GetInstanceID(dst network.Address, partyID []byte) (ol.InstanceID, error) {
+	si := &network.ServerIdentity{Address: dst}
+	ret := &GetInstanceIDReply{}
+
+	err := c.SendProtobuf(si, &GetInstanceID{partyID}, ret)
+	return ret.InstanceID, err
+}
+
+// StoreSigner asks the service to store an Signer for a given party.
+func (c *Client) StoreSigner(dst network.Address, partyID []byte, Signer darc.Signer) error {
+	si := &network.ServerIdentity{Address: dst}
+	ret := &StoreSignerReply{}
+
+	return c.SendProtobuf(si, &StoreSigner{partyID, Signer}, ret)
+}
+
+// GetSigner asks the service for an Signer for a given party.
+func (c *Client) GetSigner(dst network.Address, partyID []byte) (darc.Signer, error) {
+	si := &network.ServerIdentity{Address: dst}
+	ret := &GetSignerReply{}
+
+	err := c.SendProtobuf(si, &GetSigner{partyID}, ret)
+	return ret.Signer, err
+}
```

Comment:
These are new client APIs, but it seems they're not tested. Can we have some simple tests for these to make sure they work?